torch/_inductor/bounds.py

```diff
@@ -75,7 +79,14 @@ def swap_submodules(self, submodules):
 
         return result
 
-    def masked_subblock(self, subblock, env, mask, value, submodules):
+    def masked_subblock(
+        self,
+        subblock: LoopBodyBlock,
+        env: Dict[torch.fx.Node, ValueRanges],
+        mask: Any,
+        value: Any,
+        submodules: Dict[str, Callable[..., Any]],
+    ):
```

Comment:
```suggestion
    ) -> ValueRanges:
```

Reply:
> Some of these annotations are very informative, but others are... uf. Not a big fan of those others, but well.

@lezcano Are you commenting about the state of the types effectively already in use here or do you mean we could do better, e.g., get more detailes with the Callables? If you think we can do better, I'm up for trying harder here. I _think_ many of these were what I gathered via monkeytype, but I could certainly try again.

Reply:
Oh, no, it's not your fault, it's just that many of these are rather verbose and they "do not add much information" (I'm looking at you `Callable[..., Any]`. Nothing actionable though.

I'm sorry if it sounded the wrong way :)

Reply:
Oh no problem at all, just wanted to make sure I understood!